modules/social_features/social_group/social_group.module

```diff
@@ -1925,6 +1925,15 @@ function social_group_entity_base_field_info_alter(&$fields, EntityTypeInterface
         'weight' => 16,
       ])
       ->setDisplayConfigurable('form', TRUE);
+
+    $entity_types = ['node', 'user'];
```

Comment:
@dmytriikaiun Could you add a code comment of why we are removing those fields here and make the commit message more described as well?